client/src/app/tabs/cloud-bpmn/BpmnEditor.js

```diff
@@ -0,0 +1,754 @@
+/**
```

Comment:
Except for element templates, namespace migration and plugins this is an exact copy of `tabs/bpmn/BpmnEditor`. Was this a conscious decision? Are we planning to refactor this at some point? From what I see this could easily been done by inheritance plus overriding `loadTemplates`, `handleNamespace` with no-ops and overriding `createCachedState`.

Reply:
We tried that out and decided to not do it now (one refactoring at a time): https://github.com/camunda/camunda-modeler/issues/2029#issuecomment-778163752

Reply:
But would be very happy to see it refactored any time soon :-)

Reply:
This is intended indeed. We don't want to introduce another layer in this PR, as it's already quite complex:
`React.PureComponent -> CachedComponent -> ZeebeBpmnEditor`